### PR TITLE
fix: removed last Axway entries

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: docs/.vuepress/local-components/landing/Header.vue [hunk 1]
```diff
@@ -6,9 +6,9 @@
     </div>
     <div class="container mx-auto grid grid-cols-1 md:grid-cols-12 gap-4 mt-16 mb-2 px-6">
       <div class="md:col-span-7 lg:col-span-5 flex flex-col justify-center sm:text-center md:text-left landing-header-text">
-        <h1>Write in JavaScript.<br>Run native everywhere.</h1>
+        <h1>Write in JavaScript.<br>Run native everywhere.</h1>
```
Comment (cb1kenobi) [inline docs/.vuepress/local-components/landing/Header.vue]: What's at the end of the line?

Comment (caspahouzer) [reply docs/.vuepress/local-components/landing/Header.vue]: Dunno. Maybe a formatting space. Will remove it

Comment (caspahouzer) [reply docs/.vuepress/local-components/landing/Header.vue:9]: There's nothing I can remove

Bildschirmfoto 2022-05-22 um 08 36 07

Comment (caspahouzer) [reply docs/.vuepress/local-components/landing/Header.vue]: fixed

**Review by cb1kenobi — APPROVED** [review]
**APPROVED**!

**Pull request merged**